R/datamodelr-code.R

```diff
@@ -272,16 +274,25 @@ bdm_create_graph_list <- function(
       })
     }
   )
-
   g_labels <-
     sapply(names(tables), function(x) {
-      dot_html_label(
-        tables[[x]],
-        title = x,
-        palette_id = data_model$tables[data_model$tables$table == x, "display"],
-        col_attr = col_attr,
-        columnArrows = columnArrows
-      )
+      if (legacy) {
```

Comment:
Can we add a `legacy` argument to the `dot_html_label()` function, and from there forward as necessary?